_config.yml

```diff
@@ -18,10 +18,10 @@ email: msse@berkeley.edu
 description: >- # this means to ignore newlines until "baseurl:"
   A website for the MSSE Bootcamp
 repository: 
-baseurl: "" # the subpath of your site, e.g. /blog
+baseurl: "" /blog-alpha-chi-omega
 url:  # the base hostname & protocol for your site, e.g. http://example.com
 twitter_username: 
-github_username:  
+github_username: waynexuchem
```

Comment:
Don't add your github username here. It will be different for everyone and will cause conflicts when merging to the central repository. You can remove this and make a new commit.